### ISSUE TITLE
Directed Donations

### ISSUE THREAD

**Issue by coke** [tpf/www.perlfoundation.org#14]
We used to have targeted donations, where someone could request their funds go against a particular item (e.g. perl 5 maintenance, perl 6 development). Looking just now, I see only a single donate page that doesn't have a selection or a comment.

How can someone do this today, and can we add a mechanism for this on the donation page itself?

**Comment by coke** [comment]
tagging @dwright

**Comment by dwright** [comment]
You can donate to Perl 5 CMF here:   https://perlfoundation.z2systems.com/np/clients/perlfoundation/donation.jsp?campaign=2

We haven't set up a URL for Perl 6, but could if somebody wanted it.

**Comment by coke** [comment]
Yes please! I'll update the donate page with both links when ready.
